Explorer/Assets/DCL/Chat/_Refactor/ChatChannels/ChatChannelsPresenter.cs

```diff
@@ -67,6 +71,8 @@ public ChatChannelsPresenter(ChatChannelsView view,
             this.chatEventBus.OpenPrivateConversationRequested += OnOpenUserConversation;
             this.chatEventBus.OpenCommunityConversationRequested += OnOpenCommunityConversation;
 
+            this.communityDataService.OnCommunityMetadataUpdated += OnOnCommunityChannelMetadataUpdated;
```

Comment:
Can you please rename it to remove these `On` in names - one from event itself and another from function:
`communityDataService.CommunityMetadataUpdated  += OnCommunityChannelMetadataUpdated;`